CRM/Core/xml/Menu/Admin.xml

```diff
@@ -580,7 +580,7 @@
   <item>
      <path>civicrm/admin/options/grant_type</path>
      <title>Grant Types</title>
-     <desc>List of types which can be assigned to Grants. (Enable CiviGrant from Administer > Systme Settings > Enable Components if you want to track grants.)</desc>
+     <desc>List of types which can be assigned to Grants. (Enable CiviGrant from Administer > System Settings > Enable Components if you want to track grants.)</desc>
```

Comment:
This change has already been merged to core in your other PR, so if you rebase this one it will disappear:
`git pull --rebase origin master`